examples/react/react-threejs/src/App.tsx

```diff
@@ -0,0 +1,30 @@
+import { ThreeGrid } from "./ui/components/Three";
+import "./App.css";
+import { useMemo } from "react";
+import { KeyboardControlsEntry, KeyboardControls } from "@react-three/drei";
+import { UIContainer } from "./ui/components/UIContainer";
+
+export enum Controls {
```

Comment:
let's move this to dedicated file somewhere

Reply:
I removed this, it is never used